openraft/src/membership/membership.rs

```diff
@@ -357,6 +357,14 @@ impl<C: RaftTypeConfig> Membership<C> {
         Ok(m)
     }
 
+    pub(crate) fn remove_members(&self, members: BTreeSet<C::NodeId>) -> BTreeSet<C::NodeId> {
+        self.get_configs().last().cloned().unwrap().difference(&members).cloned().collect::<BTreeSet<_>>()
+    }
+
+    pub(crate) fn add_members(&self, members: BTreeSet<C::NodeId>) -> BTreeSet<C::NodeId> {
+        self.get_configs().last().cloned().unwrap().union(&members).cloned().collect::<BTreeSet<_>>()
+    }
+
```

Comment:
These two methods can be removed now. Otherwise, the method names should be updated to reflect their actual behavior.